src/index.d.ts

```diff
@@ -5170,6 +5203,138 @@ declare namespace admin.messaging {
   }
 }
 
+declare namespace admin.machineLearning {
+    /**
+     * Interface representing options for listing Models.
+     */
+    interface ListModelOptions {
+      listFilter?: string;
+      pageSize?: number;
+      pageToken?: string;
+    }
+
+    /** Response object for a listModels operation. */
+    interface ListModelsResult {
+      readonly models: Model[];
+      readonly pageToken?: string;
+    }
+
+    /**
+     * A TFLite Model output object
+     */
+    interface TFLiteModel {
+      readonly sizeBytes: number;
+
+      readonly gcsTfliteUri?: string;
+    }
+
+    /**
+     * A Firebase ML Model input object
+     */
+    interface ModelOptions {
+      displayName?: string;
+      tags?: string[];
+
+      tfLiteModel?: {gcsTFLiteUri?: string;};
```

Comment:
I think `gcsTFLiteUri` should be required.

Reply:
Sure. We can require it for now. It is the only option in a oneof. Eventually there will be more options. Then they will all be optional. (But at least one will be required.)